src/test/mir-opt/lower_intrinsics.wrapping.LowerIntrinsics.diff

```diff
@@ -30,7 +30,7 @@
           _4 = _1;                         // scope 0 at $DIR/lower_intrinsics.rs:7:45: 7:46
           StorageLive(_5);                 // scope 0 at $DIR/lower_intrinsics.rs:7:48: 7:49
           _5 = _2;                         // scope 0 at $DIR/lower_intrinsics.rs:7:48: 7:49
--         _3 = wrapping_add::<T>(move _4, move _5) -> bb1; // scope 0 at $DIR/lower_intrinsics.rs:7:14: 7:50
+-         _3 = std::intrinsics::wrapping_add::<T>(move _4, move _5) -> bb1; // scope 0 at $DIR/lower_intrinsics.rs:7:14: 7:50
```

Comment:
I added this to make the lower_intrinsics test pass but I'm not really familiar with this. I'm guessing that now MIR needs the newly added `std::intrinsics::` to disambiguate from the new trait methods, but I'm not sure about this. (Same for the two other cases below.)